rules/S6453/java/metadata.json

```
@@ -0,0 +1,18 @@
+{
+  "title": "Region bound definitions should be consistent",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "javadoc"
```

Let's add Java 18 tag

```suggestion
    "javadoc",
    "java18"
```